### PR TITLE
Update visual style of plain files in the timeline

### PR DESCRIPTION
**Pull request by turt2live**
----

Note: The voice message rendering here is for reference purposes only. It is updated in https://github.com/matrix-org/matrix-react-sdk/pull/5970

----

image
image

### CODE REVIEW

**Review by gaelledel — APPROVED** [review]
thanks!

File: res/themes/dark/css/_dark.scss [hunk 1]
```diff
@@ -213,9 +213,10 @@ $breadcrumb-placeholder-bg-color: #272c35;
 
 $user-tile-hover-bg-color: $header-panel-bg-color;
 
-$message-body-panel-bg-color: #21262c82;
-$message-body-panel-icon-bg-color: #8e99a4;
-$message-body-panel-fg-color: $primary-fg-color;
+$message-body-panel-fg-color: $secondary-fg-color;
+$message-body-panel-bg-color: $voice-record-waveform-bg-color;
```
Comment (jryans) [inline res/themes/dark/css/_dark.scss:217]: Surely `$voice-record-waveform-bg-color` is "more specific", and so the actual colour value might go here, and then `$voice-record-waveform-bg-color` might be set to `$message-body-panel-bg-color` (inverting what you have here)?

**Review by jryans — APPROVED** [review]
A small note on CSS variables, but overall looks good.

File: res/themes/dark/css/_dark.scss (hunk 1 above)
Comment (turt2live) [reply res/themes/dark/css/_dark.scss:217]: hmm, yea. I have somewhat been debating whether or not after https://github.com/matrix-org/matrix-react-sdk/pull/5970 lands if we should consolidate styles, but the components are just slightly too different to be the same...

Comment (turt2live) [reply res/themes/dark/css/_dark.scss:217]: switching the variable order around is blocked on https://github.com/matrix-org/matrix-react-sdk/pull/5970

**Review by jryans — APPROVED** [review]

**Pull request merged**